scripts/prepare/index.ts

```diff
@@ -30,6 +30,19 @@ await Promise.all(
   ),
 );
 
+console.log('Generating components list package.json resources.');
+
+const components = await readdir(local.components);
```

Comment:
Wouldn't it be simpler to create two URLs — like `local.components.lit` and `local.components.react`? Or even `local.litPackageJson` and `local.reactPackageJson`? We could avoid unnecessary IO operation (`readdir`).

Reply:
We also need their respective destinations, then there would be 4 URLs in total. Not any simpler IMO.

To avoid the IO operation, I've replaced `readdir` with the `lit`/`react` options exported form the config.